luatest/server.lua

```diff
@@ -396,6 +397,7 @@ end
 -- following string: `Failed to copy artifacts for server (alias: <alias>, workdir: <workdir>)`.
 function Server:save_artifacts()
     if self.artifacts_saved then
+        log.verbose('Server %s artifacts already saved to %s', self.alias, self.artifacts)
```

Comment:
`Artifacts of server %s already saved to %s`

Reply:
Fixed